lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -0,0 +1,16 @@
+module SqlFootprint
+  class SqlAnonymizer
+    GSUBS = {
+      /\sIN\s\((.*)\)/ => ' IN (values-redacted)'.freeze, # IN clauses
+      /\s\=\s([0-9]+)/ => ' = number-redacted'.freeze, # numbers
```

Comment:
Is it safe to exclude the `=` like we did with string values? I imagine this will fail to catch `<` and `>`.